account_reconcile_model_multicompany_propagate/views/account_reconcile_model.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+
+
+    <record
+        id="account_reconcile_model_inherit_account_reconcile_model_multicompany_propagate"
+        model="ir.ui.view"
+    >
+        <field
+            name="name"
+        >account.reconcile.model.inherit.account_reconcile.model.multicompany.propagate</field>
+        <field name="model">account.reconcile.model</field>
+        <field name="inherit_id" ref="account.view_account_reconcile_model_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//form/sheet" position="before">
+                <header>
+                    <button
+                        name="propagate_multicompany_account_reconcile_model"
+                        icon="fa-clone"
+                        string="Propagate to companies"
+                        title="Propagate to companies with same chart of accounts"
+                        type="object"
+                        groups="account.group_account_user"
```

Comment:
IMHO it'd be good to confirm before proceeding:

```suggestion
                        groups="account.group_account_user"
                        confirm="This will copy this model to other companies where you have access, if they share the same chart of accounts. If they already have a model with the same name, it will be updated in place. Are you sure to proceed?"
```